HW Switch.txt

```diff
@@ -0,0 +1,28 @@
+using System;
+
+class MainClass {
+  public static void Main (string[] args) {
+
+    Console.WriteLine("What is 3 + 5?");
+	Console.WriteLine("A) 5");
+    Console.WriteLine("B) 6");
+    Console.WriteLine("C) 7");
+    Console.WriteLine("D) 8");
+	string question =	Console.ReadLine();
```

Comment:
Align the line and delete the space between `=` and `Console.ReadLine();`
Also change the name to `answer`, the user input is the answer to the question.